tested/languages/utils.py

```diff
@@ -33,6 +36,12 @@ def jvm_cleanup_stacktrace(traceback: str,
 
         # skip line if not a new File line is started
         if context_file_regex.search(line):
+            if unresolved_main_regex.search(line):
```

Comment:
Couldn't this be a `contains` call instead of a regex search?

Reply:
Yes, I will change it